tools/gfx-unit-test/gfx-test-util.cpp

```diff
@@ -202,6 +202,14 @@ namespace gfx_test
         const char* searchPaths[] = { "", "../../tools/gfx-unit-test", "tools/gfx-unit-test" };
         deviceDesc.slang.searchPathCount = (SlangInt)SLANG_COUNT_OF(searchPaths);
         deviceDesc.slang.searchPaths = searchPaths;
+
+        gfx::D3D12DeviceExtendedDesc extDesc = {};
+        extDesc.rootParameterShaderAttributeName = "root";
```

Comment:
Should we go ahead and codify the `[root]` attribute and turn it into a feature directly supported by Slang? It sure seems that at the point where we need dedicated support for it in `gfx`, we might as well just consider it a Slang/`gfx` feature.